target/linux/ipq40xx/image/generic.mk

```diff
@@ -814,8 +814,7 @@ define Device/netgear_ex6100v2
 	DEVICE_MODEL := EX6100
 	DEVICE_VARIANT := v2
 endef
-# Missing DSA Setup
-#TARGET_DEVICES += netgear_ex6100v2
+TARGET_DEVICES += netgear_ex6100v2
```

Comment:
A few lines further down, the EX6150 should be uncommented as well, this patch applies to both EX6100 and EX6150